portfolio/src/main/webapp/index.html

```diff
@@ -58,7 +58,7 @@ <h1>Gallery</h1>
         <section class="container">
           <div class="gallery">
             <a target="_blank" href="images/remarkables.jpg">
-              <img class="galleryImg" src="images/remarkables.jpg">
+              <img class=galleryImg src="images/remarkables.jpg">
```

Comment:
I don't think you need quotation marks around the class for this to work, but it is [recommended by the style guide](https://google.github.io/styleguide/htmlcssguide.html#HTML_Quotation_Marks). Did you remove them for a reason?

Reply:
Not sure why it was deleted. I'll put them back.